packages/app/src/backup/services/backup-create.service.test.ts

```diff
@@ -158,7 +162,27 @@ describe("Backup process failed path", () => {
   test("Returns the `Result.failed` with `BackupError.CannotReachBackupLocation` if `DeviceInfo` endpoint return error status", async () => {
     deviceManager.device.request = jest
       .fn()
-      .mockResolvedValue(Result.failed(new AppError("", "")))
+      .mockImplementation((config: { endpoint: Endpoint; method: Method }) => {
+        if (
+          config.endpoint === Endpoint.Backup &&
+          config.method === Method.Post
+        ) {
+          return Result.failed(new AppError("", ""))
+        }
+
+        return Result.failed(new AppError("", ""))
+      })
```

Comment:
If I get it right, it always return the same (`Result.failed(new AppError("", ""))` - so the `if` is unnecessary here